src/App.test.tsx

```diff
@@ -7,49 +7,47 @@ import App from "./App";
 describe("<App />", () => {
   it("navigates to 'About' page after clicking on it on header", async () => {
     testRenderer(<App />);
-    const aboutPageNavLink = screen.getByText(/O metodzie/i);
+    const aboutPageNavLink = screen.getByText("aboutMethodPageLink");
     userEvent.click(aboutPageNavLink);
 
     expect(screen.getByText("Co to są Myślące Klasy?")).toBeInTheDocument();
   });
 
   it("navigates to 'Moderator' page after clicking on it on header", async () => {
     testRenderer(<App />);
-    const moderatorPageNavLink = screen.getByText(/Strefa moderatora/i);
+    const moderatorPageNavLink = screen.getByText("moderatorPageLink");
     userEvent.click(moderatorPageNavLink);
 
     expect(screen.getByText("Witaj, moderatorze!")).toBeInTheDocument();
   });
 
   it("navigates to main page after clicking on the logo", async () => {
     testRenderer(<App />);
-    const moderatorPageNavLink = screen.getByText(/Strefa moderatora/i);
+    const moderatorPageNavLink = screen.getByText("moderatorPageLink");
     userEvent.click(moderatorPageNavLink);
-    const logo = screen.getAllByAltText(/Logo aplikacji/i);
+    const logo = screen.getAllByAltText("logoAlt");
     userEvent.click(logo[0]);
 
-    expect(
-      screen.getByText("Szukaj problemów matematycznych")
-    ).toBeInTheDocument();
+    expect(screen.getByText("inputHeader")).toBeInTheDocument();
```

Comment:
Here also we need to change `"inputHeader" `for `"searchInputHeader"`

Reply:
Thanks!